### ISSUE TITLE
♿ 🐞 Bug Report: Elementor 3.13.x - Font color same as background color in Elementor UI Select fields

### ISSUE THREAD

**Issue by anmolmehrotra** [elementor/elementor#22430]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

In the custom code section, the font color and the background color for Display condition section is the same making it invisible to the user.
![image](https://github.com/elementor/elementor/assets/14567560/4fdf1dbe-927e-4100-8485-ff9e16a279cc)

The same happens when the drop down menu is selected for include or exclude options
![image](https://github.com/elementor/elementor/assets/14567560/ea765801-8628-4226-ad1f-00085f91fa1c)

The bug is only there for include and exclude options and looks fine for the display options beside it.

### Steps to reproduce

1. In the Elementor section on the left on WordPress admin panel, select Custom code
2. Click on edit beside Conditions option
3. Click on the drop down menu for include or exclude display options

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details><summary>System Info</summary>
<p>

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.12
	PHP Version: 8.0.28
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: [Private]
	Home URL: [Private]
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 5.5
	Admin Email: admin@[Private].co.in
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.68

== Active Plugins ==
	Conditionally display featured image on singular pages and posts
		Version: 2.14.0
		Author: Cyrill Bolliger

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	Fluent Forms
		Version: 4.3.25
		Author: Contact Form - WPManageNinja LLC

	Fluent Forms Pro Add On Pack
		Version: 4.3.23
		Author: Fluent Forms

	Spectra
		Version: 2.5.1
		Author: Brainstorm Force

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast

== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active
	Hello Theme Header &amp; Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default

== Log ==
	
Log: showing 20 of 352023-05-09 08:25:05 [info] elementor-pro::elementor_pro_updater Started 
2023-05-09 08:25:05 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-09 08:25:05 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-09 08:25:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.0',
)]
2023-05-09 08:25:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.0',
)]
2023-05-10 00:00:13 [info] elementor::elementor_updater Started 
2023-05-10 00:00:13 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-10 00:00:13 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-10 00:00:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-10 00:00:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-12 01:08:56 [info] elementor-pro::elementor_pro_updater Started 
2023-05-12 01:08:56 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-12 01:08:56 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-12 01:08:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-12 01:08:56 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-12 01:12:44 [info] elementor::elementor_updater Started 
2023-05-12 01:12:44 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-12 01:12:44 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-12 01:12:44 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-05-12 01:12:44 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]

PHP: showing 4 of 4PHP: 2023-04-27 22:24:42 [warning X 43][/home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-03 02:04:37 [warning X 1][/home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(84): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor/data/base/endpoint.php(166): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> update_item()
#3: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor/data/base/endpoint.php(307): Elementor\Data\Base\Endpoint -> base_callback()
#4: /home/u246436969/domains/[Private].co.in/public_html/test/wp-includes/rest-api/class-wp-rest-server.php(1181): Elementor\Data\Base\Endpoint -> Elementor\Data\Base\{closure}()
',
)]
PHP: 2023-05-03 02:05:27 [warning X 20][/home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/u246436969/domains/[Private].co.in/public_html/test/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-05-04 02:08:15 [warning X 46][/home/u246436969/domains/[Private].co.in/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: /home/u246436969/domains/[Private].co.in/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/u246436969/domains/[Private].co.in/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/u246436969/domains/[Private].co.in/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/u246436969/domains/[Private].co.in/public_html/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]

JS: showing 4 of 4JS: 2023-05-02 19:07:52 [error X 1][https://staging.[Private].co.in/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:632658] Cannot read properties of undefined (reading &#039;isDesignable&#039;) 
JS: 2023-05-02 19:55:56 [error X 22][https://staging.[Private].co.in/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:302910] Cannot read properties of undefined (reading &#039;id&#039;) 
JS: 2023-05-02 19:57:14 [error X 1][https://staging.[Private].co.in/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:1081263] Cannot read properties of null (reading &#039;getBoundingClientRect&#039;) 
JS: 2023-05-02 20:48:43 [error X 1][https://staging.[Private].co.in/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:785234] Cannot read properties of undefined (reading &#039;toLowerCase&#039;) 

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

</p>
</details>

**Comment by nicholaszein** [comment]
Hello @anmolmehrotra! 

Thank you for submitting your issue! 🙏 

We are continually working on improving our UI following our recent Facelift of the editor, and there are still some parts that will receive some upgrades. We understand this is not ideal but we are aware of this issue and will release fixes in the future. We ask for patience as we work to fix these issues.

Kind regards

**Comment by toineenzo** [comment]
Also seeing this black title on dark background on some sites. It seems to be random
<img width="331" alt="image" src="https://github.com/elementor/elementor/assets/4921397/ec27b015-71ef-46d2-b76e-70d16510f174">

**Comment by sinanisler** [comment]
please fix this asap CANT USE DARK MODE anymore...  😥

**Comment by sinanisler** [comment]
@toineenzo   in 14 beta it is fixed I didnt look in deeply but most of the fields are good in dark mode now.

**Comment by rami-elementor** [comment]
The bugs mentioned in this issue are fixed. Tested with Elementor 3.14.1. Closing the ticket.
